### PR TITLE
new code to process delta files

### PR DESCRIPTION
**Pull request by dho937**

### CODE REVIEW

**Comment by dho937** [comment]
@karlmdavis   This is my initial approach to processing the delta files.  I still need to do more unit testing but wanted to give you a chance to look at first.  Thanks.

**Comment by dho937** [comment]
@karlmdavis Will do.

File: bluebutton-data-model-rif/src/main/resources/db/scripts/Add_Final_Action_Column.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/*
+ * This script will add the finalAction column to the below tables with a default of "F"
+ */
+
+
+-- Add finalAction column to below tables.
+
+  
+alter table "CarrierClaims" 
+   add column "finalAction" char(1) not null default "F";
```
Comment (karlmdavis) [inline bluebutton-data-model-rif/src/main/resources/db/scripts/Add_Final_Action_Column.sql]: Just a note: we'll eventually need to get the columns ordered correctly. That's a problem with all the other columns already there, though, so oh well.

File: bluebutton-data-model-rif-samples/src/main/java/gov/hhs/cms/bluebutton/data/model/rif/samples/StaticRifResourceGroup.java [hunk 2]
```diff
@@ -22,7 +22,9 @@
 
 	SAMPLE_C(StaticRifResource.SAMPLE_C_BENES, StaticRifResource.SAMPLE_C_CARRIER, StaticRifResource.SAMPLE_C_INPATIENT,
 			StaticRifResource.SAMPLE_C_OUTPATIENT, StaticRifResource.SAMPLE_C_SNF, StaticRifResource.SAMPLE_C_HOSPICE,
-			StaticRifResource.SAMPLE_C_HHA, StaticRifResource.SAMPLE_C_DME, StaticRifResource.SAMPLE_C_PDE);
+			StaticRifResource.SAMPLE_C_HHA, StaticRifResource.SAMPLE_C_DME, StaticRifResource.SAMPLE_C_PDE),
+
+	SAMPLE_U(StaticRifResource.SAMPLE_U_BENES, StaticRifResource.SAMPLE_U_CARRIER);
```
Comment (karlmdavis) [inline bluebutton-data-model-rif-samples/src/main/java/gov/hhs/cms/bluebutton/data/model/rif/samples/StaticRifResourceGroup.java]: You don't have any ITs that use this new sample delta data, best as I can tell. Can you add a new test case that verifies they work as expected?

Comment (karlmdavis) [reply bluebutton-data-model-rif-samples/src/main/java/gov/hhs/cms/bluebutton/data/model/rif/samples/StaticRifResourceGroup.java]: Gosh darned it. I have -- yet again -- gotten confused by the split between `bluebutton-data-model` and `bluebutton-data-pipeline`. My apologies: I see `RifLoaderIT.updateSampleU()` now. Sigh.

Comment (dho937) [reply bluebutton-data-model-rif-samples/src/main/java/gov/hhs/cms/bluebutton/data/model/rif/samples/StaticRifResourceGroup.java]: no worries...I created a couple more tests to ensure the updates were working correctly.

**Pull request merged**